app/views/giveaways/index.html.erb

```diff
@@ -24,20 +24,22 @@
     <%else %>
         <div class="columns is-mobile is-multiline is-left">
             <% @giveaways.each do |giveaway|%>
-            <div class="box has-background-white-ter" style="margin-bottom: 24px">
-                <%= link_to giveaway_path(giveaway), class:"card" do %>
-                    <p class="bd-notification is-primary">
-                        <small><%= l(giveaway[:created_at], format: :short) %></small><br>
-                        <code class="html"><small><%= giveaway.location %></small></code><br>
-                        <h2><%= giveaway.title.truncate(20)%></h2>
-                        <% unless giveaway.pictures.empty? %>
-                            <div class="img image is-128x128">
-                                <%= image_tag(giveaway.pictures.first.image) %>
-                            </div>
-                        <%end%>
-                    </p>
+                <% unless !giveaway.approved_to.nil?%>
```

Comment:
instead of double negation you could use 
`if giveaway.approved_to.nil?` instead

Reply:
Yes it's simpler. I will change it.